packages/next/src/build/swc/options.ts

```diff
@@ -183,6 +183,7 @@ export function getJestSWCOptions({
   esm,
   nextConfig,
   jsConfig,
+  resolvedBaseUrl,
   pagesDir,
   hasServerComponents,
 }: // This is not passed yet as "paths" resolving needs a test first
```

Comment:
You can remove this comment

Reply:
Removed in https://github.com/vercel/next.js/pull/45815/commits/cdeba5a5990d4b97629c1634ddfe56efff4394c0